data/src/main/java/org/oppia/data/backends/gae/model/GaeClassroomTopicSummarytDict.kt

```diff
@@ -0,0 +1,21 @@
+package org.oppia.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class GaeClassroomTopicSummarytDict(
+
+  @Json(name = "version") val version: Int?,
+  @Json(name = "id") val id: String?,
+  @Json(name = "name") val name: String?,
+  @Json(name = "language_code") val language_code: String?,
```

Comment:
We don't need language code. Remove this

Reply:
Ok removed